### DISCUSSION TITLE
Is whatsapp-web.js a possible alternative for Whatsapp via Twilio?

### DISCUSSION THREAD

**Discussion by FelipeGlauber** [chatwoot/chatwoot#3259] [Feature Requests]
A brazilian startup called UTalk (https://utalk.umbler.com/) makes use of this project to sell a service of Whatsapp with multi-agent feature via Puppeteer.

Twilio didn't approve my Whatsapp request. Is this a possible alternative in future?

https://github.com/pedroslopez/whatsapp-web.js/

**Comment by sojan-official** [comment]
Apart from Twilio we also support 360dialog at the moment, you can take a look at them. 
About our plans on adding other providers, please refer : https://github.com/chatwoot/chatwoot/discussions/2759

**Reply by FelipeGlauber** [reply]
Thanks @sojan-official. By the way, congratulation! Chatwoot is an awesome product!

**Reply by luisdanielhj** [reply]
Hi, I have a client requesting me to integrate whatsapp-web.js with Chatwoot self hosted is this posible?
